packages/x-date-pickers/src/internals/components/PickersArrowSwitcher/PickersArrowSwitcher.types.tsx

```diff
@@ -16,6 +16,11 @@ export interface ExportedPickersArrowSwitcherProps {
    */
   slotProps?: PickersArrowSwitcherSlotsComponentsProps;
   classes?: Partial<PickersArrowSwitcherClasses>;
+  /**
+   * Format used to display the date.
+   * @default `${adapter.formats.month} ${adapter.formats.year}`
+   */
+  format?: string;
```

Comment:
I did not add any prefix on this prop name, but I can if you want

Reply:
I think this is ok as is ... or would you expect a change in the future that would conflict with another property?

Reply:
We name the toolbar prop `toolbarFormat`, that's why I'm unsure about the best name here.

But I also think `format` is clear enough.

Reply:
Works for me! 💪🏻